### DISCUSSION TITLE
Multiple Sites

### DISCUSSION THREAD

**Discussion by Newbiect** [iptv-org/community#510] [Ideas]
### Description

It would be interesting to add an option to be able to extract information from different sites at the same time in a single xml file.

Something like this:

```
  <channels>
    <channel lang="en" xmltv_id="3ABNEnglish.us" site_id="three-angels-broadcasting-network/19066">3ABN English</channel>
    <channel lang="en" xmltv_id="5StarMaxEast.us" site_id="5-star-max-eastern/636">5 StarMax East</channel>
    <channel lang="en" xmltv_id="5StarMaxWest.us" site_id="5-star-max-pacific/2627">5StarMax West</channel>
    <channel lang="en" xmltv_id="DisneyXDEast.us" site_id="9100001138#9200004852">Disney XD East</channel>
    <channel lang="en" xmltv_id="DIYNetwork.us" site_id="9100001138#9200012415">DIY Network USA</channel>
    <channel lang="es" xmltv_id="Atreseries.es" site_id="ar#atreseries">Atreseries</channel>
    <channel lang="es" xmltv_id="AzCorazon.mx" site_id="ar#az-corazon">Az Corazón</channel>
  </channels>
</site>
```

**Comment by PopeyeTheSai10r** [comment]
How would the program know which site to extract data from when a xmltv_id is available from multiple sources?

**Comment by Newbiect** [comment]
> How would the program know which site to extract data from when a xmltv_id is available from multiple sources?

I suppose that by telling the program from which sites to extract, it is not necessary to extract an entire site if you only need the information from several channels. I am just saying an idea but of course I don't know if it is something possible to do.

**Comment by freearhey** [comment]
This feature was added in [patch 2023.10.1](https://github.com/iptv-org/epg/pull/2177).

More info: https://github.com/iptv-org/epg?tab=readme-ov-file#use-custom-channel-list

**Reply by Newbiect** [reply]
Thank you so much, they are amazing guys.
